### ISSUE TITLE
dulicate entries in optd_por_public

### ISSUE THREAD

**Issue by wsteitz** [opentraveldata/opentraveldata#20]
There are some duplicate entries:
ZZZ^WRLE
SBD

In the SBD case, I am expecting two entries because it is contained in the "no longer valid" and in the "best known" file. But it just appears twice with the same geonames ID

WRLE is one of those ICAO-only airports. While it appears twice, another one from the "best known" file is missing.

**Comment by wsteitz** [comment]
WRLE is no longer duplicated

**Comment by wsteitz** [comment]
great. Seems to be resolved with the latest updates.

**Comment by denisarnaud** [comment]
Yes, SBD was a hard one!
SBD now designates (again) San Bernardino, CA, USA, and SBT (which used to be San Bernardino for a few years) now designates Sabetta, Russia.

**Comment by denisarnaud** [comment]
Thanks Steve for your suggestion!

> As of today, SBT is still showing as the IATA code for San Bernadino in optd_por_public.csv lines 14107 and 14110, and as Sabetta in lines 14108 and 14109. Here is line 14107

Yes, indeed, the two records for San Bernardino, CA, US, have a non empty envelope ID, _i.e._, they are no longer active records from the IATA perspective.

As a reminder, the beginning of the header (with the field names) is as following:
`iata_code^icao_code^faa_code^is_geonames^geoname_id^envelope_id^name^asciiname^latitude^longitude^fclass^fcode^page_rank^date_from^date_until^comment`

When the envelope ID is non empty (no longer active records for IATA; those records are curated in the [`opentraveldata/optd_por_no_longer_valid.csv` file](https://github.com/opentraveldata/opentraveldata/blob/master/opentraveldata/optd_por_no_longer_valid.csv), where as the active records are curated in the [`opentraveldata/optd_por_best_known_so_far.csv` file](https://github.com/opentraveldata/opentraveldata/blob/master/opentraveldata/optd_por_best_known_so_far.csv)), the `date_until` is non empty too, and states from which date the record became inactive.
The `comment` field gives more explanations on what happened. In the case of SBT, the end date of activity/validity is 31 December 2014, and the comment reads:

> The IATA code has changed to SBD (Geonames ID of the city is still equal to [5391710](http://geonames.org/5391710))

In other words, you can filter out the inactive records by keeping only the records having an empty `envelope_id` field.
